common/lib/xmodule/xmodule/course_module.py

```diff
@@ -663,7 +663,14 @@ class CourseFields(object):
         # Ensure that courses imported from XML keep their image
         default="images_course_image.jpg"
     )
-
+    issue_badges = Boolean(
+        display_name=_("Issue OpenBadges"),
+        help=_(
+            "Issue OpenBadges badges for this course. Badges are generated when certificates are created."
```

Comment:
Do we want to explicitly mention "OpenBadges" here or keep it generic in case another badging service is used?